cryptoapis/README.md

```diff
@@ -16,10 +16,11 @@ The adapter takes the following environment variables:
 - `base`, `from`, or `coin`: The symbol or ID of the coin to query
 - `quote`, `to`, or `market`: The symbol or ID of the market to convert to
 
-### Difficulty endpoint
+### Blockchain stats
```

Comment:
"Blockchain stats" is a non-existing concept. 

We should document input params for `difficulty` and `height` endpoints (data points). Something like this:

```markdown
## Input Params

- `endpoint` or `dataPoint`: The requested data point. One of (`price`|`difficulty`|`height`). Defaults: `price`.

### Price

- `base`, `from`, or `coin`: The symbol or ID of the coin to query (required).
- `quote`, `to`, or `market`: The symbol or ID of the market to convert to (required).

### Difficulty

- `blockchain` or `coin`: The blockchain name (required).
- `network` or `chain`: The blockchain network name. Default: `mainnet`

### Height

- `blockchain` or `coin`: The blockchain name (required).
- `network` or `chain`: The blockchain network name. Default: `mainnet`
```

I hope this makes it more clear now. I've introduced new names in this example like `dataPoint` & `chain`, just to  make it complete and more understandable. Please do not include those in the docs as we do not support them currently.

Reply:
Added Difficulty & Height separately, along with sample responses.